src/main/app/src/app/formulieren/formulieren.module.ts

```diff
@@ -1,22 +1,29 @@
 /*
- * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-FileCopyrightText: 2024 Dimpact
```

Comment:
If we are strict I guess this should really be:

```suggestion
 * SPDX-FileCopyrightText: 2021 Atos, 2024 Dimpact
```

That's also how we do it whenever we change existing code (but then we use `Lifely` as author).

Reply:
Aangepast.

Reply:
thanks!